application/src/main/java/org/opentripplanner/standalone/config/buildconfig/TransferParametersMapper.java

```diff
@@ -0,0 +1,44 @@
+package org.opentripplanner.standalone.config.buildconfig;
+
+import static org.opentripplanner.standalone.config.framework.json.OtpVersion.V2_7;
+
+import org.opentripplanner.graph_builder.module.TransferParameters;
+import org.opentripplanner.standalone.config.framework.json.NodeAdapter;
+
+public class TransferParametersMapper {
+
+  public static TransferParameters map(NodeAdapter c) {
+    TransferParameters.Builder builder = new TransferParameters.Builder();
+    builder.withMaxTransferDuration(
+      c
+        .of("maxTransferDuration")
+        .summary("This overwrites the default `maxTransferDuration` for the given mode.")
+        .since(V2_7)
+        .asDuration(TransferParameters.DEFAULT_MAX_TRANSFER_DURATION)
+    );
+    builder.withCarsAllowedStopMaxTransferDuration(
+      c
+        .of("carsAllowedStopMaxTransferDuration")
+        .summary(
+          "This is used for specifying a `maxTransferDuration` value to use with transfers between stops which are visited by trips that allow cars."
+        )
+        .description(
+          """
+This parameter configures additional transfers to be calculated for the specified mode only between stops that have trips with cars.
+The transfers are calculated for the mode in a range based on the given duration.
+By default, these transfers are not calculated for the specified mode.
+"""
+        )
```

Comment:
Since this parameter is rather odd on the first glance, you could explain a bit here that this might be useful when there are ferries that sort of extend the road network.

Reply:
Yeah, it's a difficult concepts so needs a bit more docs.